torch/testing/_internal/common_utils.py

```diff
@@ -69,6 +69,116 @@
 IS_FBCODE = os.getenv('PYTORCH_TEST_FBCODE') == '1'
 IS_REMOTE_GPU = os.getenv('PYTORCH_TEST_REMOTE_GPU') == '1'
 
+
+class _TestParametrizer(object):
+    """
+    Decorator class for parametrizing a test function, yielding a set of new tests spawned
```

Comment:
For language consistency the decorated function is the test "template" which is instantiated into the actual tests

Reply:
Follow-up on this, this is really the base class to inherit from if writing such a decorator, right? A pointer to an example of a class that inherits from this in the docs here would be neat, too